images/virtualization-artifact/pkg/controller/service/ip_address_service.go

```diff
@@ -29,18 +30,18 @@ import (
 
 type IpAddressService struct {
 	logger      logr.Logger
```

Comment:
Do you need logger for this service?

Reply:
Indeed, we write logs only in the constructor. We can opt out of member.